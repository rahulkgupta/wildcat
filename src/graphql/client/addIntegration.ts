const addIntegrationMutation = (data: any, service: string): string => {
  return `mutation MyMutation($data: jsonb = ${data}, $service: String = "${service}") {
    insert_integrations_one(object: {data: $data, service: $service}) {
      data
    }
  }
  `;
};

export default addIntegrationMutation;
