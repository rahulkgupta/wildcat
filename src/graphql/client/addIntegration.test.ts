import addIntegration from './addIntegration';

describe('addIntegration', () => {
  it('works', () => {
    expect(
      addIntegration(
        {
          key: 'value',
        },
        'service',
      ),
    ).toMatchSnapshot();
  });
});
