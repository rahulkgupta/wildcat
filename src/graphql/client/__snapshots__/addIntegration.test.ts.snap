// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`addIntegration works 1`] = `
"mutation MyMutation($data: jsonb = [object Object], $service: String = \\"service\\") {
    insert_integrations_one(object: {data: $data, service: $service}) {
      data
    }
  }
  "
`;
